packages/flutter/test/widgets/default_text_style_test.dart

```diff
@@ -40,4 +40,56 @@ void main() {
     expect(text.overflow, TextOverflow.fade);
     expect(text.maxLines, 3);
   });
+
+  testWidgets('AnimatedDefaultTextStyle changes propagate to Text', (WidgetTester tester) async {
+    const Text textWidget = const Text('Hello', textDirection: TextDirection.ltr);
+    const TextStyle s1 = const TextStyle(
+      fontSize: 10.0,
+      fontWeight: FontWeight.w800,
+      height: 123.0,
+    );
+    const TextStyle s2 = const TextStyle(
+      fontSize: 20.0,
+      fontWeight: FontWeight.w200,
+      height: 1.0,
+    );
+
+    await tester.pumpWidget(const AnimatedDefaultTextStyle(
```

Comment:
If we ever change default values of the non-animated properties, this test might silently stop testing what it intended to test. You could explicitly specify the initial values here so it does not happen.

Reply:
i'll add expects to verify the initial values are as expected